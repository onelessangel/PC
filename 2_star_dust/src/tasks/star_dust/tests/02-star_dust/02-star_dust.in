10
8 1251A284 2A46FD06 3C6FAF1B 3C628424 25D50D39 01675FBA 7027A28A 3765F7FA 
14 63FC628D 44944AA2 2E90E65E 1D228394 210C34F1 5F402768 5BB6005D 0685A3A9 12A26E4B 46B0D268 6EA6D07E 75381039 067299AE 3FC4D42A 
9 49088FA8 238B915E 4AAFAFD4 6DD4544F 698B6613 5A280402 7823C27C 7BDD0897 046F0108 
3 383F8CBB 2A440E42 35FAD151 
49 61AA063C 3BCE49C8 0C6391D3 263E50DF 6A5F3026 29861567 474A85D0 499F578E 053C15C4 4DD02979 5C41C5D9 4BECE82C 3C76F9F8 5179D613 525F81DB 7C3BCE22 7B302CCC 1B681183 1FC75F80 45DFDCA0 093C65D2 0952C594 2007E0A3 0160284E 052FCE2B 2476E1AB 35F399E5 3D6F5AE7 4EBAEFED 6BEE6B37 65D68A2C 3064F62A 27BCB4FF 723A1BFF 56A34709 121BE525 1BC03166 1DEDCCD9 5BBB3CB4 20FC472A 6BBDF652 37FD028D 6CE92F57 2834F04A 0976D8A0 3F48B132 2470BE6C 04A7056D 5AB0C2B5 
15 4A86E20D 63ED2887 4D8AE381 6A8EC2B0 654D50D6 52BAB1AC 0F05A45C 1B40EABB 102A0C93 5DC09449 072F55F2 760096C0 0E258A73 2EEC0AF1 683AB2BF 
28 4107F017 03FAE426 02B69E55 1CC32CCB 24F72B50 6E7494A8 54C02F58 11E05AA7 16A984F2 5E3707F9 51290BD9 3B1A435F 62DE0D66 2BD9CE8E 7F52614C 2D64EF73 0FC6F716 4CDD44CD 17F3B224 751447EC 1F97F679 26F95680 105532A7 2FC2030D 04B9EAC9 1784889A 25C299CD 12DF753D 
23 0DFD4C8C 77A846B9 077883A2 11F830B2 7A5EE50F 243BB06D 36EF5C03 68D379B7 78FBDFC6 48CFB6AA 7F7CFEA9 5732E7BF 19F8C284 3A974208 3A10F525 45D29112 39E9A354 6775E498 55998828 06C6E821 7F6996BC 4AADD014 265EDE9B 
18 5B0302BC 5620E1A8 2B1CD806 72878B56 7BE37B75 3DFC4D43 38F81EE1 09E0C801 35A493FC 4070A284 1BD8F8B4 3003790B 64AC52F1 52C854B7 18D6F2C2 5DA832B7 1B980B61 1853F16C 
40 3590CDE5 52EB3374 6EEC0F9B 7B635EF8 0CD4D6C9 5661F434 50FCE720 139BBEEA 55CB8AF0 1BAAB735 39FA9D85 7C2E782D 76ADB9F1 101B7F2D 274B5033 69354547 0BFEFAA2 65479D76 222D6428 15DFC2A4 1AEC3172 629E06AC 31B8BB58 4AEFAA7E 474A599E 0481100F 63C69D40 24F28C55 20191B70 7C1A8EAC 59CDA6CC 55A9E956 4F05C221 48B9B667 510D484E 5BDA98EA 1F1BAA9B 220A2F6E 6F7657D4 74E7358C 
49
S C 8 7
M S 6 36 000016B5
M C 4 74 00000079
D S 3 2
S S 6 15
D C 7 11
M I 3 1 452A73FF
S I 4 19
S C 4 38
S I 0 6
S S 9 2
M I 0 10 399F65F4
M I 7 31 0C9429F6
S S 4 47
M I 6 13 3B8E720D
D S 0 4
D I 7 17
M S 7 6 00003E04
M I 0 15 4E972003
D C 1 11
M S 1 2 00000234
M I 8 22 213F508D
M S 8 31 00004CEA
S I 4 29
S C 1 11
S C 8 8
S I 5 3
D I 7 6
S C 5 2
S I 9 19
D I 5 11
M C 9 47 00000045
D S 2 1
D C 9 8
M S 4 89 00004B17
S C 0 4
S I 3 1
M C 8 25 00000057
M C 5 27 00000048
S I 4 9
M I 7 35 2C4689BC
S S 8 5
D S 4 47
M I 0 15 561EEFC9
S I 3 2
M S 1 1 00000DA6
M S 0 8 00004132
M I 9 30 0993D119
S S 6 3
