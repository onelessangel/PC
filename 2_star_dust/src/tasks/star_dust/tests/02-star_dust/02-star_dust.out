task 1
12.98557692
task 2
1251A284 0000FD06 3C6FAF1B 41328424 25D50D39 01675FBA 8AA22770 3765F7FA 00000000 399F65F4 00000000 00000000 00000000 00000000 561EEFC9 
02340DA6 44944AA2 2E00E65E 1D228394 210C34F1 5F402768 5BB6005D 0685A3A9 12A26E4B 46B0D268 6EA6D07E 75381039 067299AE 3FC4D42A 
49080000 238B915E 4AAFAFD4 6DD4544F 698B6613 5A280402 7823C27C 7BDD0897 046F0108 
452A73FF 420E442A 51D1FA35 
61AA063C 3BCE49C8 0C6391D3 263E50DF 6A5F3026 29861567 474A85D0 499F578E 053C15C4 7929D04D 5C41C5D9 4BECE82C 3C76F9F8 5179D613 525F81DB 7C3BCE22 7B302CCC 1B681183 1FC77980 A0DCDF45 093C65D2 0952C594 2007E0A3 60010000 052FCE2B 2476E1AB 35F399E5 3D6F5AE7 4EBAEFED 376BEE6B 65D68A2C 3064F62A 27BCB4FF 723A1BFF 56A34709 121BE525 1BC03166 1DEDCCD9 5BBB3CB4 20FC472A 6BBDF652 37FD028D 6CE92F57 2834F04A 09764B17 3F48B132 2470BE6C 04A7056D 5AB0C2B5 
4A86E20D 63ED2887 4D8AE381 B0C28E6A 654D50D6 52BAB1AC 0F48A45C 1B40EABB 102A0C93 5DC09449 00000000 760096C0 0E258A73 2EEC0AF1 683AB2BF 
4107F017 FA03E426 02B69E55 1CC32CCB 24F72B50 6E7494A8 54C02F58 E0115AA7 16A984F2 5E3707F9 51290BD9 3B1A435F 3B8E720D 2BD9CE8E 7F52614C 2D64EF73 0FC6F716 16B544CD 17F3B224 751447EC 1F97F679 26F95680 105532A7 2FC2030D 04B9EAC9 1784889A 25C299CD 12DF753D 
0DFD4C8C 77A846B9 3E0483A2 11F830B2 7A5EE50F 00000000 36EF5C03 68D379B7 78FBDFC6 48CFB6AA 7F7CFEA9 5732E7BF 19F8C284 3A974208 3A10F525 45D29112 00000000 6775E498 55998828 06C6E821 7F6996BC 4AADD014 265EDE9B 00000000 00000000 00000000 00000000 00000000 00000000 00000000 0C9429F6 00000000 00000000 00000000 2C4689BC 
5B0302BC 5620E1A8 1C2BD806 72878B56 7BE37B75 3DFC4D43 38F81E57 09E0C801 35A493FC 4070A284 1BD8F8B4 3003790B 64AC52F1 52C854B7 18D6F2C2 5DA84CEA 1B980B61 1853F16C 00000000 00000000 00000000 213F508D 
3590CDE5 00EB7433 6EEC0F9B 7B635EF8 0CD4D6C9 5661F434 50FCE720 139BBEEA 55CB8AF0 1BAAB735 39FA9D85 7C45782D 76ADB9F1 101B7F2D 274B5033 69354547 0BFEFAA2 65479D76 222D6428 A4C2DF15 1AEC3172 629E06AC 31B8BB58 4AEFAA7E 474A599E 0481100F 63C69D40 24F28C55 20191B70 0993D119 59CDA6CC 55A9E956 4F05C221 48B9B667 510D484E 5BDA98EA 1F1BAA9B 220A2F6E 6F7657D4 74E7358C 
task 3
0 1 14
